python/paddle/fluid/contrib/tests/test_image_classification_fp16.py

```diff
@@ -0,0 +1,301 @@
+#   Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
This file is duplicated?
Can add arguments to the exists unittest instead of a new one?

Reply:
'Cause this interface is in `contrib`, we'd better use a seperate unit test file